### PR TITLE
Escape control characters

### PR DESCRIPTION
**Pull request by abaranch**

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
:shipit:

File: Test/CoreSDK.Test/Shared/TelemetryClientTest.cs [hunk 1]
```diff
@@ -832,17 +832,19 @@ public void AllTelemetryIsSentWithDefaultSamplingRate()
...
-            DependencyTelemetry telemetry3 = new DependencyTelemetry();
-            ExceptionTelemetry telemetry4 = new ExceptionTelemetry();
-            MetricTelemetry telemetry5 = new MetricTelemetry();
-            PageViewTelemetry telemetry6 = new PageViewTelemetry();
-            PerformanceCounterTelemetry telemetry7 = new PerformanceCounterTelemetry();
-            RequestTelemetry telemetry8 = new RequestTelemetry();
-            SessionStateTelemetry telemetry9 = new SessionStateTelemetry();
-            TraceTelemetry telemetry10 = new TraceTelemetry();
-            AvailabilityTelemetry telemetry11 = new AvailabilityTelemetry();
+            string unicodeString = "русский\\#/\x0000\x0001\x0002\x0003\x0004\x0005\x0006\x0007\x0008\x009Farabicشلاؤيثبلاهتنمةىخحضقسفعشلاؤيصثبل c\n\r\t";
```
Comment (dnduffy) [inline Test/CoreSDK.Test/Shared/TelemetryClientTest.cs]: Do we need to run PoliCheck for other languages now?

**Comment by dnduffy** [comment]
:+1:

File: Test/CoreSDK.Test/Shared/TelemetryClientTest.cs (hunk 1 above)
Comment (abaranch) [inline Test/CoreSDK.Test/Shared/TelemetryClientTest.cs]: I do not think so. We still do not support localization and those 2 strings in test are not bad.

**Pull request merged**